### PR TITLE
interface JSON edits

### PR DESCRIPTION
**Pull request by dr-frmr**

### CODE REVIEW

File: con/lib/zigs-interface-types.hoon [hunk 1]
```diff
@@ -52,19 +54,48 @@
...
+  ::
+  ++  push-cord
     ^-  cord
     '''
     [
       {"to": "ux"},
       {"amount": "ud"},
       {"from-account": "ux"},
-      {"to-account": ["unit", "ux"]}
+      {"calldata": "*"}
```
Comment (hosted-fornet) [inline con/lib/zigs-interface-types.hoon:78]: `calldata` here and in `fungible-interface-types` will fallback to `+noah`d output:

https://github.com/uqbar-dao/uqbar-core/blob/06325590e362db5d7e3d354a017ad78fae9a42e5/lib/zig/jold.hoon#L202-L213

Comment (hosted-fornet) [reply con/lib/zigs-interface-types.hoon:78]: Which is fine -- just noting it.

File: con/lib/fungible-interface-types.hoon [hunk 2]
```diff
@@ -128,7 +126,6 @@
       {
         "mints": [
           {"to": "ux"},
-          {"account": ["unit", "ux"]},
           {"amount": "ud"}
         ]
       }
```
Comment (hosted-fornet) [inline con/lib/fungible-interface-types.hoon:131]: +mint-cord here looks off to me. I think it should read:

```hoon
  ++  mint-cord
    ^-  cord
    '''
    [
      {"token-metadata": "ux"},
      {
        "mints": [
          "list",
          [
            {"to": "ux"},
            {"amount": "ud"}
          ]
        ]
      }
    ]
    '''
```

**Review by hosted-fornet — APPROVED** [review]
One comment and one thing I think is a bug. Otherwise LGTM

File: con/lib/zigs-interface-types.hoon (hunk 1 above)
Comment (dr-frmr) [reply con/lib/zigs-interface-types.hoon:78]: That's good! Should probably make this canonical with `%'*'` in the switch, even.

File: con/lib/fungible-interface-types.hoon (hunk 2 above)
Comment (dr-frmr) [reply con/lib/fungible-interface-types.hoon:131]: Thanks, yeah, will fix

**Pull request merged**